eng/pipelines/tagger.yml

```diff
@@ -93,7 +64,26 @@ steps:
     Set-Location '$(Pipeline.Workspace)/VS'
     git init
     git remote add origin https://$(System.AccessToken)@dev.azure.com/devdiv/DevDiv/_git/VS
-    git fetch --depth 1 origin $(VsCommitId)
-  displayName: Fetch VS Repo
-- powershell: . '$(Build.SourcesDirectory)/eng/scripts/CreateTagFromVSCommitTitle.ps1' -vsDirectory '$(Pipeline.Workspace)/VS/' -vsCommitId '$(VsCommitId)'
-  displayName: Create VS Insertion Tag
\ No newline at end of file
+    git fetch --depth 100 origin $(Build.SourceVersion)
+    git checkout FETCH_HEAD
+    $lastInsertionTitle = git log --pretty=%s --grep="Insert DotNet-Project-System" -1 -- :/src/ConfigData/Packages/ProjectSystem/Managed.props :/src/ConfigData/Packages/Wizard/packages.props | Select-Object -First 1
```

Comment:
Did you try using `--format` instead of `--pretty`? From memory it gives finer control over the output, which might mean you then don't need any regex matching. This looks fine as is though.

Reply:
The regex parsing is over the title we create for the PR. I assume even with format arg we can extract title at best.